### ISSUE TITLE
portuguese dict overhaul

### ISSUE THREAD

**Issue by Konfekt** [vim/vim#1318]
The portuguese spell-check dictionary is currently without maintainer and contains false positives, such as nonexistent verbs (and their conjugated forms):
```
coger, ustir, duzir, larecer sorbicar, presagiar (um 's'), microbiar,
gabiar, flogisticar, estelicar, cauteriar, buziar, fastigiar,
forniziar, topiar, linfolisar, nefanalisar, microcopiar, godiar
```
nonexistent substantives:
```
 gárria, fódia, inféria, larécia, lúzia, mólia,
parécia, quéria, ríngia, salária, gábio/a, gódia, fornízio
```
and the words
```
 blandicie, blandicia
```
are correctly 
```
 blandície, blandícia
``` 

The former maintainers have been notified, filing this serves as a reminder.

**Comment by kafeltz** [comment]
Portuguese Portugal?

**Comment by Konfekt** [comment]
No, Brazilian, the one that is automatically downloaded by `:set spelllang=pt`. Is there a continental Portuguese spellcheck dictionary for Vim?

**Comment by chrisbra** [comment]
Is this the same as #1330 ?

**Comment by Konfekt** [comment]
Thanks for pointing it out. It is a different issue, but I'll may be that @mmontu s patch has solved this problem too. @mmontu, did you update the dictionary to the current one from Libre Office?

**Comment by mMontu** [comment]
I updated it to the latest version [in extensions.openoffice.org](http://extensions.openoffice.org/en/project/vero-brazilian-portuguese-spellchecking-dictionary-hyphenator), which is from 2010. I'm not sure if it is the current one from Libre Office -- if you find a newer version I can try to update the patch using it.
